docs/site/angular-auth-oidc-client/docusaurus.config.js

```diff
@@ -12,6 +12,9 @@ module.exports = {
   favicon: 'img/favicon.ico',
   organizationName: 'damienbod', // Usually your GitHub org/user name.
   projectName: 'Angular Auth OIDC Client', // Usually your repo name.
+  markdown: {
+    format: 'md',
```

Comment:
The default parses is now `mdx2`, due to this many compile errors are thrown.
Because we don't make use of it, I changed the default parser to just `markdown`.

For example, the header in public api `forceRefreshSession(customParams?: { ... }, configId?: string)` throws an error because it tries to parse the code between the brackets `{ ... }`, because it thinks that this is a JSX block.

You can also verify this using the upgrade checker:

```
npx docusaurus-mdx-checker
```